src/js/components/tasks/cards/index.tsx

```diff
@@ -196,6 +196,11 @@ const OrgCards = ({
     }
   }, [assignUser, isWaitingToRemoveUser, orgs]);
 
+  const taskCommits = task.commits.map((c) => c.id);
```

Comment:
@jgerigmeyer I'm unclear of the setup needed to have this line covered in tests.

Reply:
Interesting... I would think probably making sure `task.commits` has at least one commit in it?